source/_docs/guides/build-tools/02-create-project.md

```diff
@@ -0,0 +1,99 @@
+    </div>
+
+    <div class="alert alert-info">
+      <h4 class="info">Note</h4>
+      <p markdown="1">Pantheon also maintains Composer based examples for [WordPress](https://github.com/pantheon-systems/example-wordpress-composer){.external} and [Drupal 7](https://github.com/pantheon-systems/example-drops-7-composer){.external} that are currently in alpha, requiring `--stability=alpha` in the command line options. While this guide demonstrates Drupal 8, the same workflow can be achieved on all frameworks.</p>
+    </div>
+
+    Provide additional information as prompted, such as:
+
+    ![Create Project Prompts](/source/docs/assets/images/pr-workflow/build-env-create-project-prompts.png)
+
+    <div class="panel panel-drop panel-guide" id="accordion">
+      <div class="panel-heading panel-drop-heading">
+        <a class="accordion-toggle panel-drop-title collapsed" data-toggle="collapse" data-parent="#accordion" data-proofer-ignore data-target="#troubleshoot-install"><h3 class="info panel-title panel-drop-title" style="cursor:pointer;"><span style="line-height:.9" class="glyphicons glyphicons-wrench"></span> Troubleshooting</h3></a>
+      </div>
+      <div id="troubleshoot-install" class="collapse" markdown="1" style="padding:10px;">
+      ### Composer Content-Length Mismatch and/or Degraded Mode
+      If you encounter an issue such as:
+
+      ```php
+      The "https://packagist.org/packages.json" file could not be downloaded: failed to open stream: Operation timed out
+      Retrying with degraded mode, check https://getcomposer.org/doc/articles/troubleshooting.md#degraded-mode for more info
+      The "https://packagist.org/packages.json" file could not be downloaded: failed to open stream: Operation timed out
+      https://packagist.org could not be fully loaded, package information was loaded from the local cache and may be out of date
+
+
+      [Composer\Downloader\TransportException]
+      Content-Length mismatch
+
+
+      create-project [-s|--stability STABILITY] [--prefer-source] [--prefer-dist] [--repository REPOSITORY] [--repository-url REPOSITORY-URL] [--dev] [--no-dev] [--no-custom-installers] [--no-scripts] [--no-progress] [--no-secure-http] [--keep-vcs] [--no-install] [--ignore-platform-reqs] [--] [<package>] [<directory>] [<version>]
+
+      [error]  Command `composer create-project --working-dir=/private/var/folders/lp/7_1gh83s5mn9lwfjvqqlf1lm0000gn/T/local-sitevPumRP pantheon-systems/example-wordpress-composer pantheon-wp-composer-project -n --stability dev` failed with exit code 1
+      ```
+
+     This indicates a network level issue related to the security settings of your modem. We recommend contacting your Internet Service Provider (ISP) and asking them to change the modem's security mode to TKIP/AES. Additionally, you can help mitigate network issues by using a non-standard channel with less activity/noise.
```

Comment:
Lots of network issues, including simply a slow connection can cause this problem. I think that the solution presented above would only help a few people.